Px.Utils.UnitTests/Validation/DataValidationTests/DataStringValueValidatorTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using PxUtils.Validation;
+using PxUtils.Validation.DataValidation;
+using System.Text;
+
+namespace Px.Utils.UnitTests.Validation.DataValidationTests
+{
+    [TestClass]
+    public class DataStringValueValidatorTest
+    {
+
+        [TestMethod]
+        [DataRow("\".\"")]
+        [DataRow("\"..\"")]
+        [DataRow("\"...\"")]
+        [DataRow("\"....\"")]
+        [DataRow("\".....\"")]
+        [DataRow("\"......\"")]
+        [DataRow("\"-\"")]
+        public void AllowedStrings(string allowedValue)
+        {
+            DataStringValidator validator = new();
+            Encoding encoding = Encoding.UTF8;
+            List<byte> value = [.. encoding.GetBytes(allowedValue)];
+            List<ValidationFeedback> feedbacks = [];
+            validator.Validate(value, EntryType.DataItem, encoding, 0, 0, ref feedbacks);
+            Assert.IsTrue(feedbacks.Count == 0);
+        }
+    
+        [TestMethod]
+        public void NotAllowedStringValue()
```

Comment:
This test should have more robust input set. Atleast following:
`"...` `..."` `...` `""` `"foo"` `".........."`